packages/automerge-repo-network-websocket/src/BrowserWebSocketClientAdapter.ts

```diff
@@ -133,16 +122,9 @@ export class BrowserWebSocketClientAdapter extends WebSocketNetworkAdapter {
   }
 
   receiveMessage(message: Uint8Array) {
-    const decoded: OutboundWebSocketMessage = CBOR.decode(new Uint8Array(message))
+    const decoded: FromServerMessage = CBOR.decode(new Uint8Array(message))
```

Comment:
not necessary now but we should probably put this through encode/decode functions which can also do validation

Reply:
(and protocol migration, etc)

Reply:
Makes sense. Once we settle on a protocol, I will separate out.